lambda-deployment-orca/src/main/java/com/amazon/aws/spinnaker/plugin/lambda/Config.java

```diff
@@ -15,4 +15,8 @@ public class Config {
     private int cacheOnDemandRetryWaitTime;
     @Value("${lambdaPluginConfig.cloudDriverPostRequestRetries:5}")
     private int cloudDriverPostRequestRetries;
+    @Value("${lambdaPluginConfig.cloudDriverRetrieveNewPublishedLambdaWait:40}")
```

Comment:
Can you make it clear in the variable name what the units are?  Also, how'd you choose the default.

Both here and the next one.

Reply:
We did lots of test for both values.
For cloudDriverRetrieveNewPublishedLambdaWait it’s the time the request for LambdaTrafficUpdateTask takes to show and start moving the weight around and the fastest that we got was 35 seconds so for cloudDriverRetrieveNewPublishedLambdaWait we decided to wait at least 40.

For cloudDriverRetrieveMaxValidateWeightsTime it’s the max time a lambda takes to finish moving the weights it was about 3 mins with 20 seconds so if the lambda is taking more than 4 minutes there is definitely something else going on and we exit the loop

Reply:
Done